client/components/Reviews/styles/Reviews.styled.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const StyledReviews = styled.div`
```

Comment:
Consider a default export instead of an in-line export for style-files that contain only one exported styled-component.